cdc/owner/feed_state_manager.go

```diff
@@ -24,6 +25,14 @@ import (
 	"go.uber.org/zap"
 )
 
+const (
```

Comment:
can you add a few comments about these default values, the basic usage and relationships among each other if necessary ?

Reply:
Added in latest commit.